packages/core/frontend/sirius-components-core/src/onboarding/NewRepresentationArea.tsx

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { gql, useMutation } from '@apollo/client';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import IconButton from '@material-ui/core/IconButton';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Snackbar from '@material-ui/core/Snackbar';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { Close as CloseIcon, Collections } from '@material-ui/icons';
+import { useEffect, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+import { NewRepresentationAreaProps } from './NewRepresentationArea.types';
+
+const useNewRepresentationAreaStyles = makeStyles(() => ({
+  card: {
+    backgroundColor: 'transparent',
+    maxWidth: '330px',
+    minWidth: '330px',
```

Comment:
- Use theme.spacing(...) instead of a hardcoded value
- This feels a bit weird, it would be better to specify the width that this component should occupy in the parent component which is responsible for layouting the children